tests/model_monitoring/test_stores/test_sql.py

```diff
@@ -50,6 +50,7 @@ def store_connection(tmp_path: Path) -> str:
         return f"sqlite:///{tmp_path / 'test.db'}"
 
     @pytest.fixture()
+    @classmethod
```

Comment:
Only this order works:
```suggestion
    @classmethod
    @pytest.fixture()
```